linera-rpc/src/grpc/client.rs

```diff
@@ -66,18 +66,24 @@ impl GrpcClient {
 
     /// Returns whether this gRPC status means the server stream should be reconnected to, or not.
     /// Logs a warning on unexpected status codes.
-    fn is_retryable(status: &Status) -> bool {
+    fn is_retryable(status: &Status, address: &str) -> bool {
         match status.code() {
             Code::DeadlineExceeded | Code::Aborted | Code::Unavailable | Code::Unknown => {
-                info!("gRPC request interrupted: {}; retrying", status);
+                info!(
+                    "gRPC request to {address} interrupted: {}; retrying",
+                    status
+                );
                 true
             }
             Code::Ok
             | Code::Cancelled
             | Code::NotFound
             | Code::AlreadyExists
             | Code::ResourceExhausted => {
-                error!("Unexpected gRPC status: {}; retrying", status);
+                error!(
```

Comment:
In my book, error-level events are things one sets a pager duty alerts to fire in the middle of a night on Sunday.